src/git.ts

```diff
@@ -232,22 +234,24 @@ export async function deploy(action: ActionInterface): Promise<Status> {
       Allows the user to specify the root if '.' is provided.
       rsync is used to prevent file duplication. */
     await execute(
-      `rsync -q -av --checksum --progress ${folderPath}/. ${
+      `rsync -q -av --checksum --progress ${action.folderPath}/. ${
         action.targetFolder
           ? `${temporaryDeploymentDirectory}/${action.targetFolder}`
           : temporaryDeploymentDirectory
       } ${
         action.clean
           ? `--delete ${excludes} ${
-              !fs.existsSync(`${folderPath}/CNAME`) ? '--exclude CNAME' : ''
+              !fs.existsSync(`${action.folderPath}/CNAME`)
+                ? '--exclude CNAME'
+                : ''
             } ${
-              !fs.existsSync(`${folderPath}/.nojekyll`)
+              !fs.existsSync(`${action.folderPath}/.nojekyll`)
```

Comment:
Will `action.folderPath` be defined here? I see it's assigned in the `init` function but I'm not sure it will be here.

Reply:
`action.folderPath` will be defined here.

```ts
const settings = {
  ...action,
  ...configuration
}


await init(settings)
status = await deploy(settings)
```

Object `settings` is passed by reference in function `init` and then passed to `deploy()`.